api/modules/publishedProjects/controller.js

```diff
@@ -45,21 +44,18 @@ class Remix {
   }
 
   _getFilesToRemix() {
-    return PublishedFilesModel.query({
-      where: {
-        published_id: this.publishedProjectsModel.get(`id`)
-      }
-    })
-    .fetchAll();
+    return Promise.fromCallback(next => {
+      return this.server.methods.publishedFilesByPublishedProject(this.publishedProjectsModel.get(`id`), next);
```

Comment:
or rather: `this.server.methods.publishedFilesByPublishedProject is not a function`